Wireframe/index.html

```diff
@@ -21,7 +21,7 @@ <h2>README file</h2>
             <p>
                 A README file is a document that gives information of a project. The information includes how to use and install the project. It can also be thought to create a first impression of a project and make the project standout.
             </p>
-            <nav><a href="Purpose of readmefile.md" title="Purpose of README file">Read more</a></nav>
+            <a href="Purpose of readmefile.md" title="Purpose of README file">Read more</a>
```

Comment:
You can't have spaces in the name of your file in `href` attribute.
I can see that your actual file name has spaces in its name. In this case you should URL-encode your spaces like this:

`<a href="Purpose%20of%20readmefile.md">`

Or I would suggest never give names to your files using spaces, get used to this notation:

purpose_of_readmefile.md

In this case your link will look like:

`<a href="purpose_of_readmefile.md">`

The same recommendation for other links with spaces in the filenames in your assignment.